lib/tasks/pacbio_data.rake

```diff
@@ -28,17 +28,15 @@ namespace :pacbio_data do
 
     print '-> Creating pacbio libraries...'
 
-    # TODO: refactor the pools array below to better correspond to requests, libraries, and pools
-    pools = [
-      { library_type: 'Pacbio_HiFi', tag_set: nil, size: 1 },
-      { library_type: 'Pacbio_HiFi', tag_set: 'Sequel_16_barcodes_v3', size: 1 },
-      { library_type: 'Pacbio_HiFi_mplx', tag_set: 'Sequel_16_barcodes_v3', size: 5 },
-      { library_type: 'PacBio_IsoSeq_mplx', tag_set: 'IsoSeq_Primers_12_Barcodes_v1', size: 1 },
-      { library_type: 'Pacbio_IsoSeq', tag_set: 'IsoSeq_Primers_12_Barcodes_v1', size: 5 },
-      { library_type: 'Pacbio_IsoSeq', tag_set: 'IsoSeq_Primers_12_Barcodes_v1', size: 5 }
+    tag_sets = [
+      {  tag_set: nil, size: 1 },
+      {  tag_set: 'Sequel_16_barcodes_v3', size: 1 },
```

Comment:
I wonder if this library_type was supposed to be associated with each request in a pool. Currently this value isn't used as library_type is generated in the reception_generator so it makes sense to remove it here. But the reception_generator gives the requests random library_types so it may not be as close to real data. Just worth noting.

Reply:
Though these aren't pools they also aren't really tagsets as the size is used to determine how many libraries/pools are made from each set. Think it might be better to remove size and just cycle through each tagset a random number of times e.g. 1..5. Thoughts?